sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/SessionOptions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Messaging.ServiceBus
+{
+    /// <summary>
+    ///
+    /// </summary>
+    public class SessionOptions
+    {
+        /// <summary>
+        ///
+        /// </summary>
+        public string SessionId { get; set; }
+
+        /// <summary>
+        ///
+        /// </summary>
+        public ServiceBusConnection Connection { get; set; }
```

Comment:
Maybe this was a detail from the design discussions that I missed and, if so, apologies but.... having the connection here feels orthogonal to the concerns of a session.   The connection itself isn't session-aware or  required to do something session-related?

Reply:
Yeah our idea was that when using sessions, you need to provide a Connection. I think this may change though so this is still pretty tentative.